.github/workflows/continous-integration.yml

```
@@ -0,0 +1,20 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Continous integration
+
+on:
+  pull_request:
+    branches: [ develop ]
+
+jobs:
+  pr-check:
+    runs-on: ubuntu-latest
```

Use fixed version to prevent surprises.

```suggestion
    runs-on: ubuntu-20.04
```